server/src/test/java/org/apache/druid/server/QueryResourceTest.java

```diff
@@ -401,8 +515,7 @@ public void run()
     startAwaitLatch.await();
 
     Executors.newSingleThreadExecutor().submit(
-        new Runnable()
-        {
+        new Runnable() {
```

Comment:
formatting, I don't know if we care about internal patches

Reply:
Good catch! Yes, it should be fixed.